.travis.yml

```diff
@@ -17,13 +17,14 @@ env:
       ALLOWED_HOSTS= "localhost"
       SENTRY_DSN=""
       REDIS_URL="redis://localhost:6379"
+      PIPENV_IGNORE_VIRTUALENVS=1
       
 
 install:
-  - pip install -r requirements.txt
+  - make setup
 
 before_script:
   - psql -c 'create database test_db;' -U postgres
 
 script:
-  - python manage.py test --settings=floodrelief.test_settings
+  - pipenv run python manage.py test --settings=floodrelief.test_settings
```

Comment:
Can you please move this command to makefile and refere it as `make test` in travis.yml?